.lintr

```diff
@@ -1,9 +1,10 @@
-linters:
-  linters_with_defaults(
-    line_length_linter = line_length_linter(100)
-  )
-exclusions:
-  c(
-    "inst/rstudio",
-    "inst/templates"
-  )
+linters:

```

Comment:
This file has line endings changed from `LF` to `CRLF`; the same happened with other files in this PR. Please use `LF` in all files consistently; LMK if you need help with that.